core/src/main/resources/application.yml

```diff
@@ -98,6 +98,8 @@ feast:
         authorizationUrl: http://localhost:8082
         subjectClaim: email
         
+    disableRestControllerAuth: false
```

Comment:
@SwampertX can we add comments documenting:
- what is this flag for?
- how does it relate to `authentication` & `authorization` flag
- Rest Controller usability if not enabled?

Reply:
It allows unauthenticated access to the REST controller. Perhaps `bypass` would be a better description?

Reply:
This is more a documentation problem than a naming problem. It is not immediately clear to the user what this flag does and when they should use it. (ie need to use `disableRestControllerAuth` in order to use rest controller cause auth is not yet completed should be documented somewhere). A comment providing usage info and some context would suffice.

Reply:
Added, please take a look @mrzzy .